sdk/src/main/java/io/savantlabs/stylus/sdk/jupyter/CondaPackage.java

```diff
@@ -0,0 +1,34 @@
+package io.savantlabs.stylus.sdk.jupyter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+import lombok.Data;
+
+@Data
+public class CondaPackage {
+  //List<String> properties; //list has 3 el -> name, version, channel
+  private String name, version, channel;
+  public boolean hasPackage(String name) throws InterruptedException {
```

Comment:
this should not be a method of `CondaPackage`. it should be a method of `JupyterClient` interface